packages/amplify-provider-awscloudformation/src/commands/awscloudformation/override.ts

```diff
@@ -3,17 +3,28 @@
 */
 
 import path from 'path';
-import { generateOverrideSkeleton, $TSContext } from 'amplify-cli-core';
+import { generateOverrideSkeleton, $TSContext, FeatureFlags } from 'amplify-cli-core';
+import { printer } from 'amplify-prompts';
+import * as fs from 'fs-extra';
 
 const subcommand = 'override';
 
 export const name = 'overrides';
 
 export const run = async (context: $TSContext) => {
-  const backendDir = context.amplify.pathManager.getBackendDirPath();
+  if (FeatureFlags.getBoolean('overrides.project')) {
+    const backendDir = context.amplify.pathManager.getBackendDirPath();
 
-  const destPath = path.normalize(path.join(backendDir, 'awscloudformation'));
-  const srcPath = path.normalize(path.join(__dirname, '..', '..', '..', 'resources', 'overrides-resource'));
-
-  await generateOverrideSkeleton(context, srcPath, destPath);
+    const destPath = path.normalize(path.join(backendDir, 'awscloudformation'));
+    const srcPath = path.normalize(path.join(__dirname, '..', '..', '..', 'resources', 'overrides-resource'));
+    // empty build dir for old CFN files
+    // no need for rollback since these files will be autogenerated after push
+    const rootStackBuildPath = path.join(destPath, 'build');
+    if (fs.existsSync(rootStackBuildPath)) {
+      fs.emptyDirSync(rootStackBuildPath);
+    }
+    await generateOverrideSkeleton(context, srcPath, destPath);
+  } else {
+    printer.info('Overrides are not enabled for Root stack . Turn overrides.project = true in cli.json');
```

Comment:
Project level overrides is currently not turned on. In cli.json file please include the following:
```
{
  override: {
     project: true
  }
}
```